README.md

```diff
@@ -13,3 +13,94 @@ flowchart LR
     RP -->|Evidence fa:fa-receipt| V[Veraison]
     V -->|EAR fa:fa-check-square| RP
 ```
+
+## Building
+
+The `keybroker-demo` has 2 executables: `keybroker-server` and `keybroker-app`.
+These are built with:
+
+```console
+$ cd rust-keybroker
+$ cargo build
+```
+
+By default, the executables are in debug mode and located in directory
+`target/debug/`.
+
+## Running
+
+The `keybroker-server` and `keybroker-app` can be controlled with command line
+options. Use `-h` / `--help` to get their usage, e.g.:
+
+```
+$ target/debug/keybroker-server --help
+A simple web service that can provide keys and secrets in exchange for verifiable attestation tokens.
+
+Usage: keybroker-server [OPTIONS]
+
+Options:
+  -a, --addr <ADDR>
+          The interface on which this server will listen (use 0.0.0.0 to listen on all interfaces) [default: 127.0.0.1]
+  -p, --port <PORT>
+          The port on which this server will listen [default: 8088]
+  -e, --endpoint <ENDPOINT>
+          The address at which this server can be reached to request a key or submit an evidence. It will be set by default to 'http://{addr}', but this value can be overridden with an FQDN for {addr} in order to use name resolution for example. The port number will be appended, so don't leave a trailing '/' to the FQDN
+      --verifier <VERIFIER>
+          The URL where the verifier can be reached [default: http://veraison.test.linaro.org:8080]
+  -m, --mock-challenge
+          Use the static CCA example token nonce instead of a randomly generated one
+  -v, --verbosity...
+          Increase verbosity
+  -q, --quiet
+          Silence all output
+      --reference-values <REFERENCE_VALUES>
+          File containing a JSON array with base64-encoded known-good RIM values [default: reference-values.json]
+  -h, --help
+          Print help
+  -V, --version
+          Print version
+```
+
+The simplest way to get started with `keybroker-server` and `keybroker-app` is
+to run them locally in _mocking_ mode (i.e they make use of statically known
+values).
+
+In a terminal, start `keybroker-server` with:
+
+```console
+$ target/debug/keybroker-server -v -m --reference-values ../testdata/rims-matching.json
```

Comment:
A more transparent way to pass the reference values is:
`--reference-values <(echo '{ "reference-values": [ "MRMUq3NiA1DPdYg0rlxl2ejC3H/r5ufZZUu+hk4wDUk=" ] }')`